### PR TITLE
v0.10.2 Add support for the funding API

### PR DESCRIPTION
**Pull request by Kenadia**

### CODE REVIEW

File: dydx/client.py [hunk 1]
```diff
@@ -765,8 +765,87 @@ def get_perpetual_markets(
...
+        Get the current and predicted funding rates.
+
+        IMPORTANT: The `current` value returned by this function is not active
+        until it has been mined on-chain, which may not happen for some period
+        of time after the start of the hour. To get the funding rate that is
+        currently active on-chain, use the get_perpetual_market() or
+        get_perpetual_markets() function.
+
+        The `current` rate is updated each hour, on the hour. The `predicted`
+        rate is updated each minute, on the minute, and may be null if no
```
Comment (BrendanChou) [inline dydx/client.py:788]: Do we want to remove `minute on the minute` if we intend to change it later?

Comment (Kenadia) [reply dydx/client.py:788]: I don't think we intend to change it -- the premium is calculated at a random time, and then the predicted rate should change at the end of the interval (on the minute)

**Comment by Kenadia** [comment]
Rebased with master

**Comment by Kenadia** [comment]
Tested locally

**Pull request merged**